src/H5FDpublic.h

```diff
@@ -123,7 +121,6 @@
 /**< Defining the H5FD_FEAT_ALLOCATE_EARLY for a VFL driver will force
```

Comment:
Should the #define come after the text documenting it?

Reply:
@lrknox I fixed it.  Thank you!